packages/router-plugin/tests/code-splitter.test.ts

```
@@ -1,12 +1,22 @@
 import { readFile, readdir } from 'node:fs/promises'
 import path from 'node:path'
 import { describe, expect, it } from 'vitest'
+import { parseAst } from '@tanstack/router-utils'
 
 import {
+  buildDeclarationMap,
+  buildDependencyGraph,
+  collectIdentifiersFromNode,
```

ESLint

[error] 'collectIdentifiersFromNode' is defined but never used.

(unused-imports/no-unused-imports)